src/dtos/collection.ts

```diff
@@ -0,0 +1,8 @@
+import { TitleDto } from "./";
```

Comment:
Idk about you, but this import always makes me a little uncomfortable. I would import directly from `./title` just to avoid potential circular dependencies (Which can cause strange side effects)